packages/js/example/index.html

```diff
@@ -35,6 +44,21 @@
       })
 
       MatomoInstance.trackEvents()
+
+      var buttonNum = 0
+      function addDynamicButton(data) {
+        buttonNum += 1
+        var button = document.createElement("button");
+        button.setAttribute("data-matomo-event", "click")
+        button.setAttribute("data-matomo-category", "sample-page")
+        button.setAttribute("data-matomo-action", "click-event")
+        button.setAttribute("data-matomo-name", "testDynamicButton")
+        button.setAttribute("data-matomo-value", buttonNum)
+        button.innerHTML = `Track Event ${buttonNum}`;
+
+        var buttonContainer = document.getElementById("buttonContainer");
+        buttonContainer.appendChild(button);
+      }
```

Comment:
Could you change the code style here to match how our linter is configured as well? E.g. no semicolons and single quotations for strings.